content/source/docs/enterprise/vcs/bitbucket-server.html.md

```diff
@@ -13,6 +13,8 @@ Note that Bitbucket Server requires both OAuth authentication and an SSH key. Th
 
 -> **Version note:** TFE supports Bitbucket Server versions 4.9.1 and newer. We do not test older versions of Bitbucket Server with TFE, and they might not work as expected.
 
+~> **Important:** TFE needs to contact your Bitbucket Server instance during setup and during normal operation. If you are using the SaaS version of TFE, this means Bitbucket Server must be internet-accessible. If you need your VCS provider to only be available on a private network, consider using a private install of TFE.
```

Comment:
If you are using Private TFE, be sure your BitBucket Server instance has network connectivity to the PTFE instance.

Reply:
ah, yeah, I guess I shouldn't take that as a given either.